.github/workflows/prettier.yml

```diff
@@ -0,0 +1,38 @@
+name: Prettier
```

Comment:
Same comment as in `analytics`: https://github.com/prezly/analytics/pull/2#discussion_r859515418

Reply:
@aspirisen please update the workflow to match [the one in theme-kit-nextjs](https://github.com/prezly/theme-kit-nextjs/blob/main/.github/workflows/lint.yml) (which uses `wearerequired/lint-action`)